### ISSUE TITLE
bad exe format

### ISSUE THREAD

**Issue by hemato** [ArghArgh200/SEDS-Setup#23]
guys can you help me ?
im use 14.04 ubuntu and 1.8 wine  
i want to create space eng server 

![bad exe](https://cloud.githubusercontent.com/assets/19358903/15267423/72d087c8-19ca-11e6-8986-4543fd5d43c1.png)

**Comment by hemato** [comment]
root@hemato:~# cd spaceengineers
root@hemato:~/spaceengineers# cd dedicatedserver64
-bash: cd: dedicatedserver64: No such file or directory
root@hemato:~/spaceengineers# cd dedicatedserver64
-bash: cd: dedicatedserver64: No such file or directory
root@hemato:~/spaceengineers# cd DedicatedServer64
root@hemato:~/spaceengineers/DedicatedServer64# wine SpaceEngineersDedicated.exe -console
err:process:create_process 64-bit application L"Z:\\root\\spaceengineers\\DedicatedServer64\\SpaceEngineersDedicated.exe" not supported in 32-bit prefix
wine: Bad EXE format for Z:\root\spaceengineers\DedicatedServer64\SpaceEngineersDedicated.exe.
root@hemato:~/spaceengineers/DedicatedServer64#

**Comment by phit** [comment]
duplicate of #22 

tldr wine needs 32bit, space eng servers are 64bit only now

**Comment by ArghArgh200** [comment]
64-bit WINE and .NET 4 do not work. Space Engineers is 64-bit only now.

**Comment by hemato** [comment]
can you tell me how i can fix this problem ?

**Comment by phit** [comment]
there is no fix, you can no longer run space engineers under linux

**Comment by hemato** [comment]
than why people waste time for try  this SEDS-Setup ??
